### PR TITLE
Update examples to new enum syntax

### PR DESCRIPTION
**Pull request by laurachevalier4**
Intellij + sed to address multiline cases and cases where the enum value isn't accessed til later.

Enum syntax change described [here](https://developers.google.com/google-ads/api/docs/client-libs/python/library-version-10?hl=en#retrieving_enums)

### CODE REVIEW

**Comment by laurachevalier4** [comment]
There are still 4 instances of "*Enum":
examples/account_management/invite_user_with_access_role.py:        "AccessRoleEnum"
examples/account_management/invite_user_with_access_role.py:            "AccessRoleEnum"
examples/remarketing/upload_conversion_adjustment.py:            "ConversionAdjustmentTypeEnum"
examples/shopping_ads/add_listing_scope.py:        "ProductTypeLevelEnum"
all doing something like `client.get_type("SomethingEnum")._pb.Something.VALUE`. Is there a correct way to update these?

**Comment by BenRKarl** [comment]
For `examples/account_management/invite_user_with_access_role.py` here's the change to make for the first instance. I'll check the others to see if they're similar:

```py
# Instead of
client.get_type("AccessRoleEnum")._pb.AccessRole.Value(access_role)

# Use
client.enums.AccessRoleEnum[access_role].value
```

**Comment by BenRKarl** [comment]
For the second occurrence in `invite_user_with_access_role.py` use:

`[e.name for e in client.enums.AccessRoleEnum]`

**Comment by BenRKarl** [comment]
For `upload_conversion_adjustment`:

1.

```py
# Old
conversion_adjustment_type_enum = client.get_type(
    "ConversionAdjustmentTypeEnum"
)

# New
conversion_adjustment_type_enum = client.enums.ConversionAdjustmentTypeEnum
```

2.
```py
# Old
conversion_adjustment_type = conversion_adjustment_type_enum._pb.ConversionAdjustmentType.Value(
    adjustment_type
)

# New
conversion_adjustment_type_value = conversion_adjustment_type_enum[
         adjustment_type
].value
```

3.
```py
# Old
and conversion_adjustment_type
== conversion_adjustment_type_enum.ConversionAdjustmentType.RESTATEMENT

# New
and conversion_adjustment_type_value
== conversion_adjustment_type_enum.RESTATEMENT.value
```

**Comment by BenRKarl** [comment]
Lastly for `examples/shopping_ads/add_listing_scope.py`:

```py
product_custom_attribute_index_enum = client.enums.ProductCustomAttributeIndexEnum

product_custom_attribute_dimension = client.get_type("ListingDimensionInfo")
product_custom_attribute = (
    product_custom_attribute_dimension.product_custom_attribute
)
product_custom_attribute.index = product_custom_attribute_index_enum.INDEX0
product_custom_attribute.value = "top_selling_products"
dimensions.append(product_custom_attribute_dimension)

product_type_level_enum = client.enums.ProductTypeLevelEnum
product_type_dimension_1 = client.get_type("ListingDimensionInfo")
product_type = product_type_dimension_1.product_type
product_type.level = product_type_level_enum.LEVEL1
product_type.value = "electronics"
dimensions.append(product_type_dimension_1)

product_type_dimension_2 = client.get_type("ListingDimensionInfo")
product_type = product_type_dimension_2.product_type
product_type.level = product_type_level_enum.LEVEL2
product_type.value = "smartphones"
dimensions.append(product_type_dimension_2)
```

File: examples/misc/upload_image_asset.py [hunk 1]
```diff
@@ -37,12 +37,10 @@ def main(client, customer_id):
...
     asset_operation = client.get_type("AssetOperation")
     asset = asset_operation.create
-    asset.type_ = client.get_type("AssetTypeEnum").AssetType.IMAGE
+    asset.type_ = client.enums.AssetTypeEnum.IMAGE
     asset.image_asset.data = image_content
     asset.image_asset.file_size = len(image_content)
-    asset.image_asset.mime_type = client.get_type(
-        "MimeTypeEnum"
-    ).MimeType.IMAGE_JPEG
+    asset.image_asset.mime_type = client.MimeTypeEnum.IMAGE_JPEG
```
Comment (BenRKarl) [inline examples/misc/upload_image_asset.py:43]: This should be `client.enums.MimeTypeEnum...`

**Review by BenRKarl — APPROVED** [review]

**Review by wihl — APPROVED** [review]

**Pull request merged**